### PR TITLE
Some improvements in logging.

### PR DESCRIPTION
**Pull request by ishan16696**
**What this PR does / why we need it**:
Some small improvements in logging of `backup-restore`.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```improvement operator
NONE
```

### CODE REVIEW

File: pkg/compressor/compressor.go [hunk 1]
```diff
@@ -80,14 +80,14 @@ func DecompressSnapshot(data io.ReadCloser, compressionPolicy string) (io.ReadCl
 	case ZlibCompressionPolicy:
 		deCompressedData, err = zlib.NewReader(data)
 		if err != nil {
-			logger.Infof("Unable to decompress: %v", err)
+			logger.Errorf("Unable to decompress: %v", err)
```
Comment (aaronfern) [inline pkg/compressor/compressor.go:83]: Can you change this message to lowercase?

File: pkg/compressor/compressor.go [hunk 2]
```diff
@@ -80,14 +80,14 @@ func DecompressSnapshot(data io.ReadCloser, compressionPolicy string) (io.ReadCl
...
 			return data, err
 		}
 
 	case GzipCompressionPolicy:
 		deCompressedData, err = gzip.NewReader(data)
 		if err != nil {
-			logger.Infof("Unable to decompress: %v", err)
+			logger.Errorf("Unable to decompress: %v", err)
```
Comment (aaronfern) [inline pkg/compressor/compressor.go:90]: Same as above

**Review by aaronfern — APPROVED** [review]
Thanks for the PR!
Just a small suggestion, otherwise looks good
/lgtm

File: pkg/compressor/compressor.go (hunk 1 above)
Comment (ishan16696) [reply pkg/compressor/compressor.go]: done

**Review by aaronfern — APPROVED** [review]

**Pull request merged**